lib/util/exec/hermit.spec.ts

```diff
@@ -16,42 +21,37 @@ describe('util/exec/hermit', () => {
   });
 
   describe('findHermitCwd', () => {
-    let localDirResult: DirectoryResult;
-    let localDir: string;
-
-    beforeEach(async () => {
-      localDirResult = await tmp.dir({ unsafeCleanup: true });
-      localDir = localDirResult.path;
-
+    beforeEach(() => {
       GlobalConfig.set({ localDir });
     });
 
-    afterEach(async () => {
-      await localDirResult?.cleanup();
-    });
-
     it('should find the closest hermit cwd to the given path', async () => {
-      await writeLocalFile('nested/bin/hermit', 'foo');
-      await writeLocalFile('bin/hermit', 'bar');
-
+      findUp.mockResolvedValue(upath.join(localDir, 'nested/bin/hermit'));
```

Comment:
```suggestion
      findUp.mockResolvedValueOnce(upath.join(localDir, 'nested/bin/hermit'));
```
use once to auto reset the return value of the mock